stories/generic-tile/generic-tile.stories.js

```diff
@@ -53,7 +52,8 @@ Neutral | (default)
 Positive | \`fd-numeric-content__scale--positive\`
 Negative | \`fd-numeric-content__scale--negative\`
 Critical | \`fd-numeric-content__scale--critical\`
-Informative | \`fd-numeric-content__scale--informative\`
+Informative | \`fd-numeric-content__scale—informative\`
```

Comment:
I guess this should be kept with `--`

Reply:
Ah yes, I write the documentation in Word and it always changes -- to m dashes (—). Thanks for catching that!

Reply:
I fixed it

Reply:
@droshev is it possible to remove "Specific Tile" and just keep Generic Tile as one document?